guides/browser-support.md

```diff
@@ -0,0 +1,171 @@
+# Browser support
+
+> A friendly guide for building cross-browser web components
```

Comment:
This is broader than just web components, right? It's about websites or apps, in general. (Applies to other uses of that phrase throughout.)

Reply:
Websites and apps are just a bunch of components put together to make a larger thing.

Reply:
Sure, but it also can be interpreted as having some technology/implementation-specific connotations. Seems like it'd be clearer to just say "websites".